bgpd/bgpd.h

```diff
@@ -865,8 +866,9 @@ typedef enum {
 /* BGP message header and packet size.  */
 #define BGP_MARKER_SIZE		                16
 #define BGP_HEADER_SIZE		                19
-#define BGP_MAX_PACKET_SIZE                   4096
-#define BGP_MAX_EXTENDED_MESSAGE_PACKET_SIZE 65535
+#define BGP_STANDARD_MESSAGE_MAX_PACKET_SIZE 4096
+#define BGP_EXTENDED_MESSAGE_MAX_PACKET_SIZE 65535
+#define BGP_MAX_PACKET_SIZE BGP_MAX_EXTENDED_MESSAGE_PACKET_SIZE
```

Comment:
BGP_EXTENDED_MESSAGE_MAX_PACKET_SIZE

Reply:
oh god I forgot to compile

Reply:
I'm just not sure about this since KA and OPEN messages should be untouched (4096).